lib/susedistribution.pm

```diff
@@ -901,7 +901,7 @@ sub activate_console {
         assert_screen "text-logged-in-$user", 60;
     }
     elsif ($type eq 'serial-ssh') {
-        serial_terminal::set_serial_prompt($user eq 'root' ? '# ' : '$ ');
+        serial_terminal::set_serial_prompt($user eq 'root' ? '# ' : '> ');
```

Comment:
What is the non-root prompt change for, exactly?

Reply:
Generally user console has > not $
https://openqa.suse.de/tests/11555360/logfile?filename=serial_terminal_user.txt
I had some failures related to that, but can't find them anymore.

Reply:
User console will have whatever prompt you pass to `set_serial_prompt()`. If you pass `$ `, it'll have `$ `. If you pass `> `, it'll have `> `. `set_serial_prompt()` changes the shell prompt by setting `$PS1`.

Reply:
But whatever will not work so well on s390x https://openqa.suse.de/tests/11567269#step/zypper_lifecycle/127
I don't see reason to change default `>` to `$`

Reply:
That failure is a mismatch between
https://github.com/dzedro/os-autoinst-distri-opensuse/blob/bc5353283bcd1647d7fc799e0316eee1ecb2d730/lib/serial_terminal.pm#L365
and
https://github.com/dzedro/os-autoinst-distri-opensuse/blob/bc5353283bcd1647d7fc799e0316eee1ecb2d730/lib/susedistribution.pm#L860